wrapper-kernel/jupyter_gap_wrapper/gap/setup.g

```diff
@@ -105,6 +105,75 @@ function(dot)
               , metadata := rec( ("image/svg+xml") := rec( width := 500, height := 500 ) ) );
 end);
 
+# TikZ draw for jupyter-gap
+BindGlobal("JUPYTER_TikZSplash",
+function(tikz)
+  local fn, header, rnd, ltx, svgfile, stream, svgdata, tojupyter;
+
+  header:=Concatenation( "\\documentclass[crop,tikz]{standalone}\n",
+                "\\usepackage{pgfplots}",
+                "\\makeatletter\n",
+                "\\batchmode\n",
+                "\\nonstopmode\n",
+                "\\begin{document}",
+                "\\begin{tikzpicture}");
+  header:=Concatenation(header, tikz);
+  header:=Concatenation(header,"\\end{tikzpicture}\n\\end{document}");
+
+  rnd:=String(Random([0..1000]));
+  fn := Concatenation("svg_get",rnd);
+
+  PrintTo(Concatenation(fn,".tex"),header);
+
```

Comment:
Or, if you need more than one file, then you can also use `DirectoryTemporary` which will create a directory that is usable for temporary files.